zul/src/main/resources/web/js/zul/sel/less/listbox.less

```diff
@@ -263,7 +263,7 @@
 	}
 	&-focus {
 		> .z-listcell {
-			background: @meshContentFocusBackgroundColor;
+			background: @hoverBackgroundColor;
```

Comment:
a strange fix to assign a `hover` variable to a `focus` style.
please check with the designer first.